CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.7.1 (2019-10-10)
```

Comment:
This would be a minor version, yes?

Reply:
I was on the fence. I saw it as an update to existing functionality. Now that I go back and read the definition of a minor update, I see there's a strong argument for minor here. Changed.